src/routes/CommunitySettings/TopicsSettingsTab/TopicsSettingsTab.js

```diff
@@ -0,0 +1,239 @@
+import cx from 'classnames'
+import { find } from 'lodash/fp'
+import { boolean, arrayOf, func, number, shape, string, object } from 'prop-types'
+import React, { Component } from 'react'
+import CreateTopic from 'components/CreateTopic'
+// import { CommunityCell } from 'components/CommunitiesList/CommunitiesList'
+import Dropdown from 'components/Dropdown'
+import Icon from 'components/Icon'
+import SingleTopicSelector from 'components/TopicSelector/SingleTopicSelector'
+import ScrollListener from 'components/ScrollListener'
+import TextInput from 'components/TextInput'
+import { TOPIC_VISIBILITY } from 'store/models/Topic'
+import { inflectedTotal } from 'util/index'
+import styles from './TopicsSettingsTab.scss'
+
+const sortOptions = [
+  { id: 'name', label: 'Name' },
+  { id: 'followersTotal', label: 'Popular' }
+]
+
+const visibilityOptions = Object.keys(TOPIC_VISIBILITY).reduce((result, option) => result.concat({ value: option, label: TOPIC_VISIBILITY[option] }), [])
+
+const TOPIC_LIST_ID = 'topic-list'
+
+export default class TopicsSettingsTab extends Component {
+  state = {
+    createTopicModalVisible: false
+  }
+
+  static propTypes = {
+    community: object.isRequired,
+    defaultTopics: arrayOf(shape({
```

Comment:
Might want to define a `topicPropType` and use it everywhere you're doing this (maybe just these two places). This line of course then becomes simply `defaultTopics: arrayOf(shape(topicPropType))`